internal/v2/s2av2_e2e_test.go

```diff
@@ -128,9 +132,33 @@ func startServer(t *testing.T, s2aAddress string, localIdentities []*commonpbv1.
 	return lis.Addr().String()
 }
 
+// startFallbackServer runs a GRPC echo testing server.
+// It's used to test the default fallback logic upon S2A failure.
+func startFallbackServer(t *testing.T) string {
+	lis, err := net.Listen("tcp", ":0")
+	if err != nil {
+		t.Errorf("net.Listen(tcp, :0) failed: %v", err)
+	}
+	cert, err := tls.X509KeyPair(serverCertpem, serverKeypem)
+	if err != nil {
+		t.Errorf("failure initializing tls.certificate: %v", err)
+	}
+	creds := credentials.NewTLS(&tls.Config{
+		Certificates: []tls.Certificate{cert},
```

Comment:
Should we hardcode the min/max TLS version in this tls.Config to 1.3 to make sure that the client is negotiating TLS 1.3?

Reply:
added thanks